person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def generate_id
+    rand(1..1000)
+  end
+end
```

Comment:
- According to the project [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#turn-your-person-class-to-nameable), please make sure your Person class inherits from Nameable. Please fix this 🙏 
- Also, according to the requirements, make sure that this class has a method correct_name implemented. It should simply return the name attribute.